DSCResources/MSFT_xDhcpServerAuthorization/MSFT_xDhcpServerAuthorization.psm1

```diff
@@ -141,9 +141,8 @@ function Get-IPv4Address
     {
         Write-Verbose $LocalizedData.ResolvingIPv4Address
         Get-WmiObject Win32_NetworkAdapterConfiguration -Namespace 'root\CIMV2' |
-            Where IPEnabled -eq 'True' |
-                ForEach-Object
-                {
+            Where-Object IPEnabled -eq 'True' |
```

Comment:
@iainbrighton 
Where is an alias to where-object.  It is always good to be explicit, but I don't see how this fixes issues #17 'xDhcpServerAuthorization fails without IP Address'  Perhaps I'm missing some history?


url

)